scout/server/blueprints/variants/views.py

```diff
@@ -299,19 +305,20 @@ def sv_variants(institute_id, case_name):
         "",
     ]
     return dict(
-        institute=institute_obj,
```

Comment:
Nice with ordered items in here

Reply:
Sure, it's not liket they get modified more!